openfl/component/ca/ca.py

```diff
@@ -27,6 +28,26 @@
 CA_CONFIG_JSON = Path('config/ca.json')
 
 
+def get_system_and_architecture():
+    """Get system and architecture of machine."""
+    uname_res = platform.uname()
+    system = uname_res.system.lower()
+
+    architecture_aliases = {
+        ('x86_64', ): 'amd64',
+        ('armv6l', ): 'armv6',
+        ('armv7l', ): 'armv7',
+        ('aarch64', ): 'arm64'
+    }
```

Comment:
Maybe better to use Dict[str, str] instead of Dict[tuple, str]

Reply:
Dict[tuple, str] was used due to suggestion that arch has multiple aliases in 'platform' module, but it was not confirmed for now.